api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 257**
+
+* [`POST /register`](/api/register-queue): Added new fields `rendered_name`
```

Comment:
This should also mention other endpoints like `GET /events` and the `GET` endpoint for custom profile fields. And in the `POST /register` response as well, these fields are added in objects of `custom_profile_fields` object as well. You can have a look at existing changes in this file to get an idea of how to write these.